datadog-monitors-monitor-handler/src/datadog_monitors_monitor/handlers.py

```diff
@@ -274,7 +274,9 @@ def build_monitor_options_from_model(model: ResourceModel) -> ApiMonitorOptions:
         if model.Options.Locked is not None:
             options.locked = model.Options.Locked
         if model.Options.NotifyAudit is not None:
-            options.notify_audit = model.Options.NotifyAudit
+            options.notify_audit = model.Options.NotifyAudit 
+        if model.Options.GroupbySimpleMonitor is not None:
+            options.groupby_simple_monitor = not model.Options.GroupbySimpleMonitor
```

Comment:
Negate the value: if CFN Multi == `true`, then groupbe_simple_monitor should == `false`